electron/electron-builder.sh

```diff
@@ -140,6 +142,7 @@ else
 fi
 
 pushd "$ARGV_BASE_DIRECTORY"
+ELECTRON_BUILDER_ARCHITECTURE="$ELECTRON_BUILDER_ARCHITECTURE" \
```

Comment:
Is this necessary?

Reply:
@zvin it needs to be exposed to build so it can interpolate the value in the `electron-builder.json` for appimage